### DISCUSSION TITLE
Importing custom query shortcuts?

### DISCUSSION THREAD

**Discussion by adman2345** [Flow-Launcher/Flow.Launcher#2630] [Q&A] [answered]
Howdy,

I'm moving from Launchy to Flow.Launcher (as are many in the near future from my company as Launchy stopped being approved) and I'm trying to convert my existing Weby shortcuts (a few hundred) into custom query shortcuts, but I don't see a means of bulk adding them, and putting them directly in the Settings.json file hasn't worked.

I have a Powershell script that will take my old config and format it the way Flow.Launcher would in the "CustomShortcuts": potion of the settings file, but I can't make it stick.

Has anyone had success with this?  I'm hoping to come up with a migration tool that will prevent folks like me from needing to build these shortcuts individually.

Thanks!

**Comment by taooceros** [comment, marked as answer]
Hi modifying the config file should work. Though you may need to close flow before modifying it or else it will be overridden by flow.

**Comment by adman2345** [comment]
Seems to be working, I missed the comma at the end of the value line, which was causing it to throw that info out.  Thanks!
